src/components/RoomView/RoomView.jsx

```diff
@@ -357,6 +394,7 @@ const RoomView = () => {
               <Item style={styles.sx.PlayerContainer}>
                 Turn:
                 {gameStatus}
+                <GuessesRemaining />
```

Comment:
Leaving a note here that this whole bar we probably will want to be its own component, not in roomview, but best to wait on this for now!